src/libmongoc/CMakeLists.txt

```diff
@@ -713,6 +712,14 @@ add_library(mongo::detail::c_dependencies ALIAS _mongoc-dependencies)
 install(TARGETS _mongoc-dependencies EXPORT mongoc-targets)
 set_property(TARGET _mongoc-dependencies PROPERTY EXPORT_NAME detail::c_dependencies)
 
+# utf8proc configuration
+find_package(Utf8Proc REQUIRED)
+if (${USE_BUNDLED_UTF8PROC})
+   list(APPEND SOURCES $<TARGET_OBJECTS:utf8proc_obj>)
+else()
+   target_link_libraries(_mongoc-dependencies INTERFACE utf8proc)
```

Comment:
If `USE_BUNDLED_UTF8PROC` is false, then the installed `_mongoc-dependencies` will contain a link reference to `PkgConfig::PC_UTF8PROC`, but that target name will not be defined for importers by default.

The following (or something close to it) will need to be added in `mongoc-config.cmake`:

```cmake
set(_mongoc_built_with_bundled_utf8proc "@USE_BUNDLED_UTF8PROC@")
if(NOT _mongoc_built_with_bundled_utf8proc AND NOT TARGET PkgConfig::PC_UTF8PROC)
  # libmongoc was compiled against an external utf8proc and links against a 
  # FindPkgConfig-genreated IMPORTED target. Find that package and generate that 
  # imported target here:
  find_dependency(PkgConfig)
  pkg_config_check_modules(PC_UTF8PROC REQUIRED utf8proc IMPORTED_TARGET GLOBAL)
endif()
```

Reply:
I added an `ALIAS` call beneath this to make sure that the generated target can also be recognized by `utf8proc`

Reply:
ALIAS targets are not installed and only create names in the build tree. When installed, all occurrences of the alias will be replaced with the aliased target name. The ALIAS target in mongoc-config.cmake is thus unnecessary.